_src/static/sass/components/_copy.scss

```diff
@@ -0,0 +1,103 @@
+// ------------------------------
+// edX Pattern Library: Components - Copy
+
+// About: Contains base styling for copy (general text and type used in content and functionaly).
+
+// #GLOBAL
+// #INDIVIDUAL COPY CASES
+
+// ----------------------------
+// #GLOBAL
+// ----------------------------
+.copy {
+    @extend %reset-copy;
+    color: $text-base-color;
+
+    &.emphasized {
+        color: $text-emphasized-color;
+        font-weight: $text-emphasized-font-weight;
+    }
+
+    &.de-emphasized {
+        color: $text-de-emphasized-color;
+        font-weight: $text-de-emphasized-font-weight;
+    }
+}
+
+// ----------------------------
+// #INDIVIDUAL COPY CASES
+// ----------------------------
+// archetypes
+%copy-lead {
+    @extend %reset-copy;
+    font-size: font-size(large);
+    line-height: line-height(mid-large);
+    color: $text-base-color;
+
+    p, ol, ul, dl {
+        margin-bottom: spacing-vertical(mid-small);
```

Comment:
NOTE: I'm not sure if we should do this here or not? Will this cause cascade/ rule bloat problems when we want to use copy elsewhere (say in a modal with its own styling for copy text or instructions above a form)?